src/system_sensors.py

```diff
@@ -141,31 +158,48 @@ def get_updates():
 
 # Temperature method depending on system distro
 def get_temp():
-    temp = "";
+    temp = ""
     if "rasp" in OS_DATA["ID"]:
-        reading = check_output(["vcgencmd", "measure_temp"]).decode("UTF-8")
-        temp = str(findall("\d+\.\d+", reading)[0])
+        reading = check_output([vcgencmd, "measure_temp"]).decode("UTF-8")
+        temp = str(re.findall("\d+\.\d+", reading)[0])
     else:
         reading = check_output(["cat", "/sys/class/thermal/thermal_zone0/temp"]).decode("UTF-8")
         temp = str(reading[0] + reading[1] + "." + reading[2])
     return temp
 
+# display power method depending on system distro
+def get_display_status():
+    display_state = ""
+    if "rasp" in OS_DATA["ID"]:
+        reading = check_output([vcgencmd, "display_power"]).decode("UTF-8")
+        display_state = str(re.findall("^display_power=(?P<display_state>[01]{1})$", reading)[0])
+    else:
+        display_state = "Unknown"
+    return display_state
+
+# host model method depending on system distro
+def get_host_model():
+    model = ""
+    if "rasp" in OS_DATA["ID"] and isDockerized:
+        model = check_output(["cat", "/app/host/proc/device-tree/model"]).decode("UTF-8").strip()
+        # remove a weird character breaking the json in mqtt explorer
+        model = model[:-1]
+    else:
+        model = "RPI "+deviceNameDisplay
```

Comment:
maybe change this to only deviceNameDisplay? because if rasp would be in OS_DATA it's a rpi otherwise it would be a different kind of device

Reply:
please see the comment below